portal/src/components/roles-and-groups/RolesAndGroupsEmptyView.tsx

```diff
@@ -1,24 +1,26 @@
-import React from "react";
+import React, { MouseEventHandler } from "react";
 import cn from "classnames";
 import { Text } from "@fluentui/react";
 import styles from "./RolesAndGroupsEmptyView.module.css";
 import PrimaryButton from "../../PrimaryButton";
 
-const CreateButton: React.VFC<{
+function CreateButton(props: {
   className?: string;
   href?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>;
   text: React.ReactNode;
-}> = function CreateButton(props) {
-  const { className, href, text } = props;
+}) {
+  const { className, href, onClick, text } = props;
   return (
     <PrimaryButton
       href={href}
+      onClick={onClick}
```

Comment:
How fluent ui handle it when both href and onClick present?

Reply:
Actually, a `<a>` element have click event too. So it is same as a `<a>` having href and onClick handler. If you preventDefault() in the onClick handler, the navigation will be stopped.